redwood-widget/src/jsMain/kotlin/app/cash/redwood/widget/HTMLElementChildren.kt

```diff
@@ -61,9 +61,11 @@ public class HTMLElementChildren(
     }
   }
 
-  override fun onLayoutModifierUpdated(index: Int) {
-    val element = parent.children[index] as HTMLElement
+  override fun onLayoutModifierUpdated() {
+    // If this function is being invoked we are guaranteed to have at least one child.
+
+    val element = parent.children[0] as HTMLElement
     parent.removeChild(element)
-    parent.insertBefore(element, parent.children[index])
+    parent.insertBefore(element, parent.children[0])
```

Comment:
Again, same behavior. We dirty the container and force it to relayout. However, unlike other platforms, we don't get a hook into layout and rely on built-in flexbox. I'm pretty sure HTML is just broken as the parent needs to be propagating modifiers into HTML attributes. But if it's broken it stays broken with this change so...